src/beamlime/config/workflow_spec.py

```diff
@@ -29,6 +28,19 @@ class WorkflowId:
     def __str__(self) -> str:
         return f"{self.instrument}/{self.namespace}/{self.name}/{self.version}"
 
+    @staticmethod
+    def from_string(workflow_id_str: str) -> WorkflowId:
+        """Parse WorkflowId from string representation."""
+        parts = workflow_id_str.split('/')
+        if len(parts) != 4:
+            raise ValueError(f"Invalid WorkflowId string format: {workflow_id_str}")
+        return WorkflowId(
+            instrument=parts[0],
+            namespace=parts[1],
+            name=parts[2],
+            version=int(parts[3]),
+        )
```

Comment:
Are these always used in the streaming? i.e. we won't save it to a file and load it later with potentially different version of beamlime?
Or when one of services start up with a new version of beamlime and pick up the latest message from the kafka broker...